### PR TITLE
Uboot cleanup

### PR DESCRIPTION
**Pull request by Ham22**
Rename to be more 'standard' and remove code that wasn't needed so was just confusing.

### CODE REVIEW

File: uboot-pistachio/Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
```
Comment (francois-berder-imgtec) [inline uboot-pistachio/Makefile]: I could

File: uboot-pistachio/Makefile [hunk 2]
```diff
@@ -17,7 +18,6 @@ define uboot/Default
   SOC:=
   DDR_SETTINGS:=
   IMAGE:=
-  DEPS:=
```
Comment (francois-berder-imgtec) [inline uboot-pistachio/Makefile]: It would be better to remove this line in the next commit

Comment (Ham22) [inline uboot-pistachio/Makefile]: Done and some other I found

File: uboot-pistachio/Makefile (hunk 1 above)
Comment (Ham22) [inline uboot-pistachio/Makefile]: Done, don't know why I added it

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins - dev

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins - dev

**Pull request merged**